roles/openshift_logging_curator/defaults/main.yml

```diff
@@ -1,7 +1,7 @@
 ---
 ### General logging settings
 openshift_logging_curator_image_prefix: "{{ openshift_hosted_logging_deployer_prefix | default('docker.io/openshift/origin-') }}"
-openshift_logging_curator_image_version: "{{ openshift_hosted_logging_deployer_version | default('latest') }}"
+openshift_logging_curator_image_version: "{{ openshift_logging_image_version }}"
```

Comment:
We lost the default.  We should retain these in case any of the roles are used separately